### ISSUE TITLE
Title menu defaulting to 'Asst. Director' in 'Loan Invoice Query1' query in Specify 7 (a11y) version

### ISSUE THREAD

**Issue by realVinayak** [specify/specify7#1825]
**Describe the bug**
The Title menu is defaulting to 'Asst. Director' in 'Loan Invoice Query1' query in Specify 7 (a11y) version. The same query in edge version has Title menu set to 'Any'. The query link is: `specify/query/20`. The username is `spbirdadmin` and the database is `Ornithology`. 

**To Reproduce**
1. Log in to a11y version of Specify 7 using username `spbirdadmin`. Select `Ornithology` database and go to 'Loan Invoice Query1' (`specify/query/20`).
2. One of the queries is Shipments -> Shipped To -> Title -> Equal -> Asst. Director
3. Repeat the above steps in the edge version and the same query is Shipments -> Shipped To -> Title (Any)

**Expected behavior**
In the a11y version, the Title should be set to 'Any' instead of 'Asst. Director'

**Screenshots**
The left part is edge version, and the right part is a11y version for the same query
![title_bug_loan_invoice_query](https://user-images.githubusercontent.com/61122018/178769916-f7d7ab84-70ad-41c2-8828-fb2d48cef8e9.PNG)

**Comment by maxxxxxdlp** [comment]
Same issue as #1823

Closed in favor of that one
